.github/scripts/execute_single_benchmark.sh

```diff
@@ -9,6 +9,26 @@ DATASET_SRC=$5
 LOADER_NAME=$6
 MAX_RESOLUTION=$7
 
+function retry {
+  local retries=$1
+  shift
+
+  local count=0
+  until "$@"; do
```

Comment:
I think this line is executing the actual command and if that command succeeds it gets the 0 signal.

Reply:
Maybe you want to document that.

Reply:
Added